http4s/src/main/scala/com/ccadllc/cedi/dtrace/http4s/client/client.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 Comcast Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ccadllc.cedi.dtrace
+package http4s
```

Comment:
I think we should consider another layer here -- e.g.:
```scala
package com.ccadllc.cedi.dtrace
package interop
package http4s
```

Same for money and xb3. Rationale is that folks often use `import com.ccadllc.cedi.dtrace._` which will than cause any subsequent imports from `http4s` to require a `_root_` prefix.